openrouteservice/src/main/java/org/heigit/ors/config/AppConfig.java

```diff
@@ -136,6 +140,16 @@ public boolean getBoolean(String path) {
 		return false;
 	}
 
+	public double getDouble(String path) {
+		try {
+			ConfigObject configObj = config.getObject("ors");
+			return configObj.toConfig().getDouble(path);
+		} catch(Exception e) {
+			// IGNORE
```

Comment:
Should we really just ignore this or should we at least write out something to an error log?

Reply:
I also thought this but didn't comment on my review, because the other functions do it the same way. So the author just copied the already existing style.

Reply:
The whole class should be refactored at some point, it's inconsistent in terms of what can be retrieved and what not, and usage patterns vary. Will leave it for now if you don't mind.

Reply:
OK, just as long as the whole error handling thing gets addressed in the future